### DISCUSSION TITLE
Read / Write PHY

### DISCUSSION THREAD

**Discussion by aspernol** [PhilipsHue/flutter_reactive_ble#340] [Q&A] [answered]
Hi,

is there a possibility to read/write PHY? I have a device here which supports it and I would like to make use of 2M PHY. Uploading data works now with 1M PHY fine but faster would be better.

Thanks and best regards,
Andreas.

**Comment by lcrocker** [comment, marked as answer]
I have created an issue about this, and also offered a StackOverflow bounty on this question:

https://stackoverflow.com/questions/69184440/replacing-a-gradle-dependency-in-flutter-library

**Reply by remonh87** [reply]
@lcrocker  did you also approach Polidea with the fix? It would be awesome to have this in so we can add this functionality
